curriculum/challenges/english/07-scientific-computing-with-python/learn-classes-and-objects-by-building-a-sudoku-solver/6606d378de78d55523f08298.md

```diff
@@ -0,0 +1,72 @@
+---
+id: 6606d378de78d55523f08298
+title: Step 41
+challengeType: 20
+dashedName: step-41
+---
+
+# --description--
+
+Now you need to check if the given number `num` is already present in the current cell of the 3x3 square.
+
+Replace `pass` with an `if` statement that checks if number in the current cell of the sudoku board is equal to `num`. Then, return `False` from the `if` body, since the number is not a valid choice.
```

Comment:
```suggestion
Replace `pass` with an `if` statement that checks if the number in the current cell of the sudoku board is equal to `num`. Then, return `False` from the `if` body, since the number is not a valid choice.
```